packages/notebook-extension/schema/tracker.json

```diff
@@ -270,7 +273,8 @@
         "readOnly": false,
         "insertSpaces": true,
         "tabSize": 4,
-        "wordWrapColumn": 80
+        "wordWrapColumn": 80,
+        "codeFolding": false
       }
     },
     "markdownCellConfig": {
```

Comment:
As with #5750, not having this option in the markdown/raw cell settings makes it unconditionally on for those cells.